docs/beamrunner/beamrunner-3-sample.md

```diff
@@ -0,0 +1,172 @@
+
+```
+validatedReadings
+        .get(badTag)
+        .apply("BadCount", Count.<String,Double>perKey())
+        .apply("BadLog", ParDo.of(new LogWindowFn<Long>("Temperature device error count")));
+```
+
+The structure of the application becomes clear after it is displayed in the Streams console after the application is run.
+
+## Running the `TemperatureSample` application
+
+1. Make sure that your Streaming Analytics service on Bluemix is running.
+2. Navigate to the `$STREAMS\_RUNNER\_HOME/samples` directory. The Streams Runner toolkit provides all necessary files. Assuming that all environment variables are set as described in Installing the toolkit and that $VCAP\_SERVICES has credentials in it named beam-service, you can launch the `TemperatureSample` application with the following command:
+
+   ```
+   java -cp \
+   $STREAMS_BEAM_TOOLKIT/lib/com.ibm.streams.beam.translation.jar:\
+   $STREAMS_RUNNER_HOME/samples/lib/com.ibm.streams.beam.samples.jar \
+   com.ibm.streams.beam.sample.temperature.TemperatureSample \
+   --runner=StreamsRunner \
+   --contextType=STREAMING_ANALYTICS_SERVICE \
+   --jarsToStage=$STREAMS_RUNNER_HOME/samples/lib/com.ibm.streams.beam.samples.jar```
+
+   Note: If the environment variables are not set, you must use full paths, and use the `--vcapServices` parameter to provide the path to the Bluemix credentials file.
+
+   For more information about the parameters available to you, see [Pipeline options for Streams Runner](../beamrunner-5-ref/#pipeline-options-for-streams-runner).
+
+3. Verify that the application started successfully.
+
+   Streams Runner displays information (INFO) messages in the terminal as it processes the Beam pipeline for your application, turning it into an IBM Streams application bundle (.sab) file, and eventually submitting the .sab file to IBM Bluemix:
+
+   ```
+   Sep 27, 2017 2:34:19 PM com.ibm.streams.beam.translation.StreamsRunner run
+   INFO: Running pipeline using StreamsRunner in context:
+        contextType=STREAMING_ANALYTICS_SERVICE
+        beamToolkitDir=null
+        tracingLevel=null
+        jobName=temperaturesample-johnmac-0927183419-7a8ccbfa
+   StreamsRunner: Sending application to be built remotely, then submitting to Streaming Analytics Service...
+   StreamsRunner: Application successfully submitted to Streaming Analytic service with Job ID: 1```
+
+   If you see an exception or error report, it indicates the reason for the failure. The following issues are common:
+
+   - A typo in the java command or one of the variables it uses.
+   - A problem with the `$VCAP\_SERVICES` file or the wrong service name. Check that the `"credentials"` for `"streaming-analytics"` in the `$VCAP\_SERVICES`file match those for your Streaming Analytics service, and that the `"name"` matches the name that you used for the `--serviceName` option to the java command.
+
+## Monitoring the application

```

Comment:
I think this section should be changed to "view the running application" since it doesn't really discuss monitoring (that's done in the next section).
Need instructions on getting to the console too.